code/game/objects/items/storage/storage.dm

```diff
@@ -506,7 +516,7 @@
 	var/turf/T = get_turf(src)
 	hide_from(usr)
 	for(var/obj/item/I in contents)
-		remove_from_storage(I, T)
+		remove_from_storage(I, T, usr)
```

Comment:
Yes I know using `usr` is bad.
Fixing this proc is outside the scope of this PR however.